### PR TITLE
Fix: failed redelegations monitor for columbus-5

### PR DESCRIPTION
**Pull request by pr0n00gler**
FailedRedelegationsMonitor is broken now due to a little bit changed response format for `/staking/delegators/{delegatorAddr}/delegations` in Columbus-5. Also this endpoint is deprecated.

So I've changed the old endpoint to a new one `/cosmos/staking/v1beta1/delegations/{delegator_addr}`

### CODE REVIEW

**Review by oopcode — APPROVED** [review]

**Comment by oopcode** [comment]
n.b.: found a potential bug with string -> uint parsing, discussed in dms, @pr0n00gler fixed it

File: collector/monitors/failed_redelegations_monitor.go [hunk 1]
```diff
@@ -78,36 +79,58 @@ func (m *FailedRedelegationsMonitor) Handler(ctx context.Context) error {
...
+			} else {
+				return fmt.Errorf("failed to get response balance: balance is nil")
+			}
+
+			tmpMetricVectors[FailedRedelegations].Set(response.Delegation.ValidatorAddress, 0)
+
+			// if delegated amount is greater than zero and the whitelisted validators don't contain a validator
+			// that means a redelegation was not successful
+			if !delegatedAmount.IsZero() && !contains(whitelistedValidators, response.Delegation.ValidatorAddress) {
+				tmpMetricVectors[FailedRedelegations].Set(response.Delegation.ValidatorAddress, 1)
```
Comment (swelf19) [inline collector/monitors/failed_redelegations_monitor.go:128]: isnt better to use monikers?

Comment (pr0n00gler) [reply collector/monitors/failed_redelegations_monitor.go:129]: The task says _New monitor is implemented using labels to show specific validator addresses on the graph_, and imo addresses are better than monikers for this panel particularly. But let's clarify this with @turinkay and @oopcode

Comment (oopcode) [reply collector/monitors/failed_redelegations_monitor.go:129]: if it's possible and not too costly, we should add a moniker in parenthesis: address (moniker)

**Review by swelf19 — APPROVED** [review]

**Pull request merged**